src/components/composed/FeeField/FeeField.js

```diff
@@ -1,25 +1,53 @@
-import React, { useCallback, useEffect, useRef, useState } from 'react'
-
+import React, {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+  useContext,
+} from 'react'
+
+import { AccountContext } from '@Contexts'
 import { InputInteger } from '@BasicComponents'
 import { OptionButtons } from '@ComposedComponents'
 import { Electrum } from '@APIs'
 import { BTC } from '@Helpers'
 
 import './FeeField.css'
 
+const ML_FEE_MOCK = JSON.stringify({
+  1: 30,
+  2: 29,
+  3: 28,
+  4: 27,
+  5: 26,
+  6: 25,
+  7: 24,
+  8: 23,
+  9: 22,
+  10: 21,
+  11: 20,
+  12: 19,
+  13: 18,
+  14: 17,
+  15: 16,
+  16: 15,
+})
+
 const FeeField = ({
   value: parentValue,
   id,
   changeValueHandle,
   setFeeValidity,
 }) => {
+  const { walletType } = useContext(AccountContext)
   const effectCalled = useRef(false)
   const [options, setOptions] = useState([])
   const [inputValue, setInputValue] = useState(0)
   const [radioButtonValue, setButtonValue] = useState(undefined)
   const [timeToFirstConfirmations, setTimeToFirstConfirmations] =
     useState('15 minutes')
   const [estimatedFees, setEstimatedFees] = useState([])
+  const feeType = walletType.name === 'Mintlayer' ? 'ML' : 'sat/B'
```

Comment:
@muursh What is going to be called the smallest part of ML? Satoshis as well? How could we express the `sat/B` for ML fees?

Reply:
So far in the core codebase we call them atoms as we don't have a better name. For now we can call them atoms and update it once there's a real term

Reply:
So @owlsua , lets replace `ML` by `atoms/B`